README.md

```diff
@@ -12,32 +31,59 @@ A nice Minecraft (non-hacked) client for 1.8.9 (and possibly newer versions in f
 - 1.7 animations
 - Item physics
 - Hypixel additions
-- OptiFine (optional, from optifine.net)
-- Crosshair
-- Quick play (command palette style) (uses games from robere2's mod)
-- Better item tooltips, showing the full description
+- OptiFine (optional, from [its website](https://optifine.net/downloads))
+- Custom crosshairs
+- Quick Play (uses games from robere2's mod)
+- Better item tooltips (like the ones in Bedrock Edition)
 - Chat symbol picker
 - Chat channel picker and status
-- Popup events (based on [this](https://github.com/Sk1erLLC/PopupEvents))
+- Popup events (based on Sk1er's [PopupEvents](https://github.com/Sk1erLLC/PopupEvents))
 - Speedometer
-- Chunk animator (based on [this](https://github.com/lumien231/Chunk-Animator))
+- Animated chunks (based on Lumien's [Chunk Animator](https://github.com/lumien231/Chunk-Animator))
 - Timers
-- Pack folders
-- Replay mod (using [this](https://github.com/ReplayMod/ReplayMod))
-- Entity culling (using an outdated version of [this](https://github.com/tr7zw/EntityCulling)!! the owner adopted a more restrictive license since which is actually fairly understandable)
+- Resource pack folders
+- Replay Mod (using the actual [Replay Mod](https://github.com/ReplayMod/ReplayMod))
+- Entity Culling (using an **outdated version** of tr7zw's [Enity Culling](https://github.com/tr7zw/EntityCulling) mod, since the owner adopted a more restrictive license)
+
+**..and even more!**
+
+## Alternatives
+
+Although Sol is not super popular by itself, here are some alternatives if you seek them:
+
+- [<img src="https://axolotlclient.github.io/images/icon.png" alt="AC icon" width="18"> AxolotlClient](https://axolotlclient.github.io/)
```

Comment:
This section feels quite empty - and most people are already aware of optifine and mod loaders.

Reply:
So I remove this?

Reply:
Hmm, maybe it is okay. 

```suggestion
## Alternatives

- [<img src="https://axolotlclient.github.io/images/icon.png" alt="AC icon" width="18"> AxolotlClient](https://axolotlclient.github.io/) is good if you wish to use versions newer than 1.8.
```

Reply:
👍🏽